### PR TITLE
feat(ui): Add ScoreCard to project detail

### PR DESCRIPTION
**Pull request by matejminar**
image

This is feature flagged (`organizations:project-detail`) work in progress.

### CODE REVIEW

**Review by jan-auer — COMMENTED** [review]
Can you please add visual snapshots for this view, so that we can also see one on the mobile viewport?

**Edit**: This then also plays into https://github.com/getsentry/sentry/pull/22118.

**Review by markstory — APPROVED** [review]

**Comment by silent1mezzo** [comment]
@k-fish is also working on something very similar. We should make sure we share these UI components in performance too.

**Comment by matejminar** [comment]
> Can you please add visual snapshots for this view, so that we can also see one on the mobile viewport?

@jan-auer I'm adding visual snapshots in a follow-up PR


> We should make sure we share these UI components in performance too.

@silent1mezzo Yep, I've added this into our storybook component library. @k-fish feel free to add props you need to tweak it to your needs.

File: src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+type Props = AsyncComponent['props'] & {};
+type State = AsyncComponent['state'] & {};
+
+class ProjectScoreCards extends AsyncComponent<Props, State> {
+  renderBody() {
+    return (
+      <CardWrapper>
+        <ScoreCard
+          title={t('Stability Score')}
+          help={t('Stability score is used to ...')}
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx:18]: help={t('Stability score is used to \u2026')}

File: src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          score="94.1%"
+          trend="+13.5%"
+          trendStyle="good"
+        />
+        <ScoreCard
+          title={t('Velocity Score')}
+          help={t('Velocity score is used to ...')}
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx:25]: help={t('Velocity score is used to \u2026')}

File: src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          score="16"
+          trend="-2 releases / 2 wks"
+          trendStyle="bad"
+        />
+        <ScoreCard
+          title={t('Apdex Score')}
+          help={t('Apdex score is used to ...')}
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx:32]: help={t('Apdex score is used to \u2026')}

File: src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import ScoreCard from 'app/components/scoreCard';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+
+type Props = AsyncComponent['props'] & {};
+type State = AsyncComponent['state'] & {};
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx:10]: type Props = AsyncComponent['props'];
type State = AsyncComponent['state'];

**Comment by priscilawebdev** [comment]
@matejminar although you have already done the merge I left some suggestions that you can apply in another pull request 😉

File: src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx (hunk 4 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx:10]: This is WIP, there will be more stuff there once API is finished

File: src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx:18]: These messages are placeholders

**Pull request merged**